assemble/src/main/resources/LICENSE

```diff
@@ -1,982 +0,0 @@
-
```

Comment:
Why is this file no longer needed? I see some changes in assembly, but not exactly sure what the big pic is.